.github/workflows/build.yml

```diff
@@ -91,10 +91,30 @@ jobs:
       run: |
         echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
         docker push antrea/antrea-ubuntu-simulator:latest
+    
 
+  check-windows-changes:
+     runs-on: [ubuntu-latest]
+     steps:
+     - uses: actions/checkout@v2
+       with:
+          etch-depth: 0
+     - name: Get changed files
+       id: changed_window_files
+       uses:  tj-actions/changed-files@v13
+       with:
+        files: |
+           *_windows.go
+           winfirewall.go
```

Comment:
this is not correct, what if there is a developer do the same thing and add a new file named `newfile.go` with the tag `//go:build windows` only in the file content?